_posts/2025-02-06-highlights-2024.md

```diff
@@ -0,0 +1,247 @@
+- Release of Bloop 2 ([v1.4.0](https://scalameta.org/metals/blog/2024/10/24/palladium/#bloop-2))
+
+## Governance
+
+One of the Scala organization’s major efforts this year was to establish, formalize, and document our own operations, as well as the policies governing releases of the Scala language, as seen on the pages linked below.
+These updated structures guide the development of Scala 3, and we’ve also documented our plans for ongoing maintenance of Scala 2.
+
+The Scala Improvement Process (SIP) also continues to operate, holding monthly meetings and leading discussions of proposed features on the Scala Contributors forum.
+In a future issue of this newsletter, we’ll highlight the committee’s handling of recent language proposals.
+
+The Scala Center‘s advisory board continues to meet quarterly.
+We were delighted to add JetBrains to the board this year.
+
+Proposals to the Center can be made directly by member companies or through the two community representatives (currently Zainab Ali and Eugene Yokota). In 2024, the following proposals were completed:
+
+- [SCP-033](https://github.com/scalacenter/advisoryboard/blob/main/proposals/033-deprecate-scala-ide.md): Deprecate Eclipse Scala IDE
+- [SCP-032](https://github.com/scalacenter/advisoryboard/blob/main/proposals/032-scala-version-guidance.md): Provide guidance on choosing between Scala LTS and Next
+- [SCP-030](https://github.com/scalacenter/advisoryboard/blob/main/proposals/030-governance-page.md): Governance page for Scala
+
+Further reading:
+
+- [Scala Governance](https://www.scala-lang.org/governance/)
+- [Scala Development Guarantees](https://www.scala-lang.org/development/)
+- [Scala 2 maintenance plans](https://www.scala-lang.org/blog/2024/12/16/scala-2-maintenance.html)
+
+## Community and events
+
+The Scala community is at the heart of Scala’s health and vitality.
+It’s exciting to see how much activity (conferences, meetups, podcasts, and more) is occurring, and we encourage everyone to participate, to grow enthusiasm around Scala.
+
+The Scala organization’s own community efforts this year included:
+
+- Google Summer of Code: 10 new contributors, 10 successful projects, as detailed in [the Scala Center page on GSoC website](https://summerofcode.withgoogle.com/programs/2024/organizations/scala-center)
+- Advent of Code, as described in [this recent blog post](https://www.scala-lang.org/blog/2025/01/16/advent-of-code-recap.html)
+
+The Center will [participate in GSoC again](https://scala-lang.org/blog/2025/01/28/gsoc-projects.html) in 2025.
+
+As always, we publish a schedule of upcoming (and past) conferences at <www.scala-lang.org/events>.
+
+Though the Scala meetup and conference scene was dealt a heavy blow by COVID, now in 2024 we see it recovering.
+We’re especially pleased to see Scala-specific conferences such as Scala.IO (Paris), Scala Matsuri (Tokyo), Scalar and Art of Scala (Warsaw), and Functional Scala (virtual) continuing.
```

Comment:
Minor note: since we're listing them, we could add links to each one:
```
We’re especially pleased to see Scala-specific conferences such as [Scala.IO](https://scala.io) (Paris), [Scala Matsuri](https://scalamatsuri.org) (Tokyo), [Scalar](https://www.scalar-conf.com) and [Art of Scala](https://artofscala.com) (Warsaw), and [Functional Scala](https://www.functionalscala.com) (virtual) continuing.
```

Reply:
I kind of intentionally didn't do that since all those same links are already at https://www.scala-lang.org/events

Reply:
in 66100aa149087ab5f5588cfbbe8803ca57b6417d I added ", with links" to emphasize that a bit